app/src/main/java/com/github/se/cyrcle/ui/theme/molecules/ReportScreenMolecules.kt

```diff
@@ -0,0 +1,122 @@
+package com.github.se.cyrcle.ui.theme.molecules
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import com.github.se.cyrcle.R
+import com.github.se.cyrcle.model.parking.ImageReportReason
+import com.github.se.cyrcle.model.parking.ParkingReportReason
+import com.github.se.cyrcle.model.report.ReportedObjectType
+import com.github.se.cyrcle.model.review.ReviewReportReason
+import com.github.se.cyrcle.ui.report.ReportScreenAlertDialog
+import com.github.se.cyrcle.ui.theme.atoms.BulletPoint
+import com.github.se.cyrcle.ui.theme.atoms.ConditionCheckingInputText
+import com.github.se.cyrcle.ui.theme.atoms.Text
+import com.github.se.cyrcle.ui.theme.disabledColor
+
+@Composable
+fun ReportTextBlock(title: String, bulletPoints: List<String>, modifier: Modifier = Modifier) {
```

Comment:
I am not really picky on documentation about composables since our code is already inconsistent but maybe for the Molecules it could be useful to have some. Since they are meant to be reused by others.

Reply:
Done! I've implemented a quick 1-line documentation detailing what the Molecules are each meant to be to make them more accessible to others